### ISSUE TITLE
Write an accessibility statement

### ISSUE THREAD

**Issue by sarahmonster** [opentexts/opentexts#150]
This may be a legal requirement, although I'm not exactly certain if this counts as a public sector website:

> If you created a new public sector website on or after 23 September 2018, you must now meet accessibility standards and publish an accessibility statement. You need to review and update your statement regularly.

Either way, it's likely a good idea to include. 

Some resources that may help:

https://www.gov.uk/government/publications/sample-accessibility-statement
https://www.accessibilitystatementgenerator.com/
https://github.com/accessibility/Accessibility-Statement
https://medium.com/@sheribyrnehaber/deconstructing-accessibility-statements-700a5f761e29

We can also use the [NLS accessibility statement](https://www.nls.uk/about-us/website-accessibility-statement) as a starting point, and adapt anything as needed.

**Comment by stuartlewis** [comment]
So, a quesiton related to this: Are there any WCAG AA requirements we don't think we meet? If we're all good, then we can use:

## Accessibility statement for OpenTexts.World

We want everyone who visits the OpenTexts.World website to feel welcome and find the experience rewarding, going above and beyond the The Public Sector Bodies Accessibility Regulations 2018.

### What are we doing?
To help us make the OpenTexts.World website a positive place for everyone, we've been using the Web Content Accessibility Guidelines (WCAG) 2.1. These guidelines explain how to make web content more accessible for people with disabilities, and user friendly for everyone.

The guidelines have three levels of accessibility (A, AA and AAA). We’ve chosen Level AA as the target for the OpenTexts.World website.

### How are we doing?
We've worked hard on the OpenTexts.World website and believe we've achieved our goal of Level AA accessibility. We monitor the website regularly to maintain this, but if you do find any problems, please get in touch.

### Let us know what you think
If you enjoyed using the OpenTexts.World website, or if you had trouble with any part of it, please get in touch. We'd like to hear from you in any of the following ways:

email us at info@opentexts.world

**Comment by sarahmonster** [comment]
This looks good to me! Yes, everything should be AA accessible, and one of my pre-launch tasks is to look through the site and make sure there's nothing important we're missing there. (See #82, on my list for this week!)
